### DISCUSSION TITLE
NIVA OXYDEP wind speed dependency

### DISCUSSION THREAD

**Discussion by Mehrnoosh-Abbasian** [fabm-model/fabm#86] [Q&A]
Hello everyone,

I am currently running GOTM-FABM and using the NIVA OxyDep model. In the `nivaoxydep.f90` file, it seems that the model might depend on wind speed as a time series, but I'm not entirely sure. Could anyone help confirm if wind speed is indeed used as a time series in this model? 
Mehrnoosh

**Comment by glessin** [comment]
Hi Mehrnoosh,

Yes, wind speed is required to calculate air-sea flux of oxygen in nivaoxydep.F90.

Cheers,
Gennadi

**Reply by Mehrnoosh-Abbasian** [reply]
THANKS FOR YOUR RESPONSE.

**Comment by jornbr** [comment]
For completeness, oxydep requests wind speed with [this line](https://github.com/fabm-model/fabm/blob/ecc1de1c6adc0ccd11f34e8cc130ec336c6168bf/src/models/niva/oxydep/oxydep.F90#L222). If you use FABM/oxydep within GOTM, it is GOTM that needs to supply the wind speed. Most GOTM setups use meteorological forcing that includes the Northward and Eastward wind components; it will then [use these to calculate the wind speed](https://github.com/gotm-model/code/blob/3c851d65688fe00bcd1c79e0d27d170f33d979e7/src/airsea/airsea.F90#L837) provided to FABM/oxydep.
